builder/role_image.go

```diff
@@ -166,19 +168,12 @@ func (r *RoleImageBuilder) CreateDockerfileDir(role *model.Role, baseImageName s
 			}
 		}
 
-		// Symlink spec configuration file
-		// from <ROOT_DIR>/opt/hcf/specs/<ROLE_NAME>/<JOB>.json
-		specConfigSource := filepath.Join(jsonSpecsDir, role.Name, job.Name+configstore.JobConfigFileExtension)
-		// into <ROOT_DIR>/var/vcap/job-src/<JOB>/config_spec.json
-		specConfigDestination := filepath.Join(jobDir, jobConfigSpecFilename+configstore.JobConfigFileExtension)
-		// The symlink must be relative, since the path is different in the docker image
-		specConfigRelativeSource, err := filepath.Rel(filepath.Dir(specConfigDestination), specConfigSource)
+		// Write spec into <ROOT_DIR>/var/vcap/job-src/<JOB>/config_spec.json
+		specConfigDestination := filepath.Join(jobDir, jobConfigSpecFilename+model.JobConfigFileExtension)
```

Comment:
Any particular reason `jobConfigSpecFileName` doesn't include the extension (now that we only have one kind anyway)?

Yes, I realize that you just copied the old code :)